src/scripts/changelog.zig

```diff
@@ -9,6 +9,8 @@ const ReleaseTriple = @import("../multiversioning.zig").ReleaseTriple;
 
 const log = std.log;
 
+const const_max_bytes = 10 * 1024 * 1024;
```

Comment:
`const_bytes_max`?

Reply:
or `const_size_max`?

Reply:
Wait, why is `const_` the prefix?

Reply:
:fearful: not sure what happened here, definitely should be `changelog_bytes_max`